### DISCUSSION TITLE
How to allow spell checking in WYSWIG?

### DISCUSSION THREAD

**Discussion by garryatbrighton** [YetiForceCompany/YetiForceCRM#15342] [How to] [answered]
I WYSWIG text editor cannot fix spelling by right-click. It is blocked. Where to fix it?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
see the ckeditor documentation 
https://ckeditor.com/docs/ckeditor4/latest/features/spellcheck.html
https://ckeditor.com/docs/ckeditor4/latest/examples/spellchecker.html 

by default, CRM using Native Browser Spell Checker
https://ckeditor.com/docs/ckeditor4/latest/features/spellcheck.html#native-browser-spell-checker

```
Note: If the Context Menu plugin is enabled, it is necessary to hold the Ctrl key when right-clicking misspelled words to see their suggestions.

Note: The spell check functionality is not available natively for all browsers.

```
